launch/baxter_planning_context.launch

```diff
@@ -0,0 +1,37 @@
+<launch>
```

Comment:
Did we actually change anything in this launch file? I think baxter_moveit.launch passes all the right arguments through, so we don't need to customize planning_context.launch. If this is the case, delete it.

Also, note that baxter_moveit.launch includes planning_context.launch from baxter_moveit_config, so this file is actually unused.